packages/components/src/systems/DraftPrescriptions/CoverageOptions/CoverageOptionSummary.tsx

```diff
@@ -0,0 +1,120 @@
+import { CoverageOption } from '../../PrescribeProvider';
+import { Banner } from '../../../index';
+import { createMemo, For, Show } from 'solid-js';
+import { BannerStatus } from '../../../particles/Banner';
+import { Prescription } from '@photonhealth/sdk/dist/types';
+import { getRefillsCount } from '../../../utils/formatRxString';
+
+export type CoverageOptionSummaryProps = {
+  coverageOption: CoverageOption;
+  prescription?: Prescription;
+};
+
+export function CoverageOptionSummary(props: CoverageOptionSummaryProps) {
+  const bannerData = createMemo<CoverageBannerData>(() => {
+    if (props.coverageOption.paRequired) {
+      return {
+        status: 'warning',
+        message: 'PA Required'
+      };
+    }
+
+    if (props.coverageOption.status === 'COVERED') {
+      return {
+        status: 'success',
+        message: 'Covered by Insurance'
+      };
+    }
+
+    if (props.coverageOption.status === 'COVERED_WITH_RESTRICTIONS') {
+      return {
+        status: 'warning',
+        message: 'Covered with Restrictions',
+        subMessage: props.coverageOption.statusMessage
+      };
+    }
+
+    return {
+      status: 'error',
+      message: 'Not Covered by Insurance',
+      subMessage: props.coverageOption.statusMessage
+    };
+  });
+
+  const usefulAlerts = createMemo(() =>
+    props.coverageOption.alerts.filter((alert) => alert.text !== props.coverageOption.statusMessage)
+  );
+
+  return (
+    <div class="flex flex-col gap-4">
+      <div class="flex flex-col">
+        <div class="text-xs text-gray-500">
+          <span>Quantity / Days Supply: </span>
+          <span class="font-bold">
+            {`${props.coverageOption.dispenseQuantity} ${props.coverageOption.dispenseUnit} / ${props.coverageOption.daysSupply}`}
+          </span>
+        </div>
+        <Show when={props.prescription}>
+          {(rx) => (
+            <>
+              <div class="text-xs text-gray-500">
+                <span>Instructions: </span>
+                <span>{rx().instructions}</span>
+              </div>
+              <div class="text-xs text-gray-500">
+                <span>Refills: </span>
+                <span>{getRefillsCount(rx().fillsAllowed)}</span>
+              </div>
+            </>
+          )}
+        </Show>
+        {/*<div class="text-xs text-gray-500">*/}
```

Comment:
can probably remove for now, I don't think this will be useful for anyone anytime soon